client/css/style.css

```diff
@@ -855,13 +854,13 @@ kbd {
 	left: 0;
 	right: 0;
 	top: 50%;
-	border-top: 1px solid #e74c3c;
+	border-top: 1px solid rgba(231, 76, 60, .5);
```

Comment:
Adds opacity, Astorije explains why in the commit message:

`This is necessary to display a non-translucide tooltip.`